tests/Application/src/Entity/SyliusInvoicingPlugin/BillingData.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Sylius\InvoicingPlugin\Application\Entity\SyliusInvoicingPlugin;
+
+use Doctrine\ORM\Mapping as ORM;
+use Sylius\InvoicingPlugin\Entity\BillingData as BaseBillingData;
+
+/**
+ * @ORM\Entity()
```

Comment:
sylius doesn't do annotation mappings, therefore sylius official plugin shouldn't as well and mappings should be converted to xml

Reply:
This is the test override entities and the test application is configured to use Annotation like in a final app when you will add your entity into your app.